### DISCUSSION TITLE
다들 코멘트는 어떻게 쓰시나요?

### DISCUSSION THREAD

**Discussion by mdgarden** [EasyAndBeauty/DailyReceipt-FE-WEB#31] [Q&A]
저는 회사 팀에서 코멘트 많이 못쓰게 하고 
/**
*
*
*/

요런식? 으로 쓰는 코멘트도 PL만 쓰지 밑에 있는 일반 사원들은 못쓰게해서 한번도 써본적이 없어요...혹시 작성법이나 간단한 팁 같은거에 대해서 알려주실 수 있나요?

**Comment by parkjooyoung0608** [comment]
저도 궁금해요!! 현업에서는 어떤식으로 작성하나요???
저는 이번에 코멘트를 처음 작성했는데, 그냥 주석으로 냅다 달았어요.. 왠지 이러면 혼날것같은...

**Reply by mdgarden** [reply]
그냥 코멘트 자체를 거의 못쓰게 해요 😥
클린코드 책에도 보면 코멘트 쓰지말라고 되어있거든요 코멘트가 있으면 그건 덜 쓴 코드라고...
근데 그건 또 영어가 모국어인 사람들의 경우지 영어권 아닌 사람들은 적절한 코멘트도 병행하면 좋을 것 같긴한데 어렵네요 😥😥😥

**Comment by IGhost-P** [comment]
저도 
/**
*
*
*/
이 방식을 많이 사용합니다! JSDoc이여서 넘 편해요!

**Reply by mdgarden** [reply]
JSDoc으로 검색하면 나오는건가요? 감사합니다!

**Comment by mdgarden** [comment]
20년차 개발자처럼 주석달기
https://www.youtube.com/shorts/FXxaVX7j9TE
